src/main/java/hudson/plugins/openid/OpenIdSession.java

```diff
@@ -87,7 +89,7 @@ public HttpResponse doCommenceLogin() throws IOException, OpenIDException {
         String url = authReq.getDestinationUrl(true);
 
         // remember this in the session
-        Stapler.getCurrentRequest().getSession().setAttribute(SESSION_NAME,this);
+        Stapler.getCurrentRequest().getSession().setAttribute(SESSION_NAME,this.toString());
```

Comment:
Yeah, [OpenIdSession.getCurrent](https://github.com/jenkinsci/openid-plugin/blob/findbugs-enable/src/main/java/hudson/plugins/openid/OpenIdSession.java#L136) expects the `OpenIdSession` instance to be stored in the session, so the above change does seem to be incompatible. Again though, I don't really see how `OpenIdSession` can be persisted properly anyway (because they don't seem to be `Serializable`).